src/tests/system/tests/test_passkey.py

```diff
@@ -26,6 +27,16 @@ def passkey_requires_root(client: Client) -> tuple[bool, str] | bool:
     return True
 
 
+@mh_fixture()
+def umockdev_ipaotpd_update(ipa: IPA, request: pytest.FixtureRequest):
+    """
+    Update the ipa-optd@.service file from ipa server
+    for umockdev
```

Comment:
Can you add more information here about why is this needed and when it is needed? I'm pretty sure we will forget it sooner or later. Otherwise ack!

Reply:
okay, let me add more, I may not forget but good to have more details description. Thank you!